src/lib/x509/asn1_alt_name.cpp

```diff
@@ -70,7 +71,9 @@ std::multimap<std::string, std::string> AlternativeName::contents() const {
    }
 
    for(const auto& othername : m_othernames) {
-      multimap_insert(names, othername.first.to_formatted_string(), othername.second.value());
+      std::string typ = othername.first.to_formatted_string();
+      std::string value = othername.second.value();
+      names.insert(std::make_pair(typ, value));
```

Comment:
```suggestion
      names.emplace(othername.first.to_formatted_string(), othername.second.value());
```

... might also save an unnecessary copy-c'tor of the `std::string` objects.